internal/network/utils.go

```diff
@@ -14,6 +15,18 @@ import (
 	"gorm.io/gorm"
 )
 
+func IsDefaultRoute(r *models.Route) (bool, error) {
+	dst := net.ParseIP(r.Destination)
```

Comment:
Isn't destination a CIDR ?

Reply:
It is a CIDR when the route is not default. At any rate this logic need to change to support both options

Reply:
From what i see default route is the one that has 0.0.0.0 as destination no?

Reply:
I added the structure as appear in the DB into the PR text. The components of the routes are IPs.

Reply:
So it can either be CIDR or IP depending if it is default or not ?

Reply:
This is a bug.  Should be fixed in the agent and all places that use it.

Reply:
But not in this PR but in a separate task